python/lsst/afw/table/baseColumnView.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import absolute_import
+
+import numpy as np
+
+from ._baseColumnView import *
```

Comment:
I'm dismayed that linter use seems to encourage otherwise-unnecessary repetition, but it sounds like we should at least have the RFC so we can get others to weigh in.  @fred3m, would you be up for putting that together?